magicblast/README.md

```diff
@@ -0,0 +1,12 @@
+# NCBI MagicBLAST docker image
+
+[NCBI MagicBLAST][1] command line applications in a Docker image.
+
```

Comment:
I would add the paper here. Should I open a pull request to your forked repository? I can also add it after your merge.

Reply:
Thanks, I can help with that also.